app/javascript/packs/homePage.jsx

```diff
@@ -54,17 +66,20 @@ function renderTagsFollowed(tagsFollowedContainer, user = userData()) {
   });
 }
 
+const feedTimeFrame = frontPageFeedPathNames.get(window.location.pathname);
+
 let waitingForDataLoad = setTimeout(function dataLoadedCheck() {
   const { user = null, userStatus } = document.body.dataset;
 
   if (userStatus === 'logged-out') {
-    // Nothing to do, the user is not logged on.
+    renderFeed(feedTimeFrame);
     return;
   }
 
   if (userStatus === 'logged-in' && user !== null) {
     // We have user data, render followed tags.
     clearTimeout(waitingForDataLoad);
+    renderFeed(feedTimeFrame);
```

Comment:
We render the feed regardless of whether the user is logged on or not, but in the case of a logged on user, we want to ensure the user data has been set before making the call to the feed endpoint.